component_combiner.js

```diff
@@ -2,7 +2,13 @@ var http = require('http');
 var port = process.env.PORT || 8000;
 var fs = require('fs');
 
-var components = ['/components/typography.html', '/components/form.html', '/components/button.html', '/components/image.html'];
+var components = [
+  '/components/typography.html',
```

Comment:
we should be able to read this from the `/components/` directory without having to _manually_ update the list. (_but it's fine for now_)